sauce/features/budget/remove-zero-categories/index.js

```diff
@@ -0,0 +1,44 @@
+import { Feature } from 'toolkit/core/feature';
+import * as toolkitHelper from 'toolkit/helpers/toolkit';
```

Comment:
it's not a big deal but I would like to move away from the pattern of bringing the entire helper in and just import what you need...

`import { getCurrentRouteName } from 'toolkit/helpers/tooklit';`